integrations/kube-agent-updater/version.go

```diff
@@ -0,0 +1,7 @@
+package kubeversionupdater
+
+const Version = "13.0.0-dev"
```

Comment:
Should `make version` be updated to keep this in sync?

Reply:
done ✅ , I will make the updater use the version var in a subsequent PR (in the user-agent and the `--version` flag)